File: Python/PagoCard.py
from pagos import Pagos

class PagoCard(Pagos):
    number = int
    ccv = int
    date = str
    name = str

    def __init__(self, id, monto, number, ccv, date, name):
        super().__init__(id, monto)
        self.number = number
        self.ccv = ccv
        self.date = date
        self.name = name

File: Java/PagoCard.java
class PagoCard extends Pagos {
    Integer number;
    Integer ccv;
    String date;
    String name;

    public PagoCard(Integer id, Integer monto, Integer number, Integer ccv, String date, String name){
        super(id, monto);
        this.number = number;
        this.ccv = ccv;
        this.date = date;
        this.name = name;
    }
}